### PR TITLE
Added parameter to set gas price for transaction to 0

### PR DESCRIPTION
**Pull request by azoumi**
Signed-off-by: azoumi <hafid.azoumi@accenture.com>

**Changelog**
-Fixed not able to do a transactions as a member node by setting min gas price to 0 as this is an enterprise network.

 
**Reviewed by**
@lakshyakumar 

 
**Linked issue**
#999

### CODE REVIEW

File: platforms/hyperledger-besu/configuration/roles/create/crypto/ibft/tasks/main.yaml [hunk 1]
```diff
@@ -189,7 +189,7 @@
   with_indexed_items: "{{ node_list }}"
   when:  
     # TODO uncomment the line below when the smartcontract can be deployed on member node.
```
Comment (lakshyakumar) [inline platforms/hyperledger-besu/configuration/roles/create/crypto/ibft/tasks/main.yaml:191]: the comment should also be removed.

**Review by lakshyakumar — CHANGES_REQUESTED** [review]
Have you checked deploying the quorum solidity smartcontract using the script on any non validating peer?

**Comment by azoumi** [comment]
> 
> 
> Have you checked deploying the quorum solidity smartcontract using the script on any non validating peer?

Yes I have tested it and it seems to deploying fine

**Review by lakshyakumar — APPROVED** [review]
Looks good to me

**Pull request merged**